.github/workflows/build-timestamped-master.yml

```diff
@@ -14,5 +14,5 @@ jobs:
   call_workflow:
     name: Run Build Workflow
     if: ${{ github.repository_owner == 'ballerina-platform' }}
-    uses: ballerina-platform/ballerina-library/.github/workflows/build-timestamp-master-template.yml@main
+    uses: ballerina-platform/ballerina-library/.github/workflows/build-timestamp-master-template.yml@java21
```

Comment:
we need to remove @java21

Reply:
Fixed in 7b4de1f18d8fc8a7c4349dfe297d09b03fa42d45